app/src/main/java/com/mxt/anitrend/adapter/recycler/detail/NotificationAdapter.kt

```diff
@@ -113,10 +113,9 @@ class NotificationAdapter(context: Context) : RecyclerViewAdapter<Notification>(
 
             binding.notificationTime.text = DateUtil.getPrettyDateUnix(model.createdAt)
 
-            if (!CompatUtil.equals(model.type, KeyUtil.AIRING) && !CompatUtil.equals(model.type, KeyUtil.RELATED_MEDIA_ADDITION)) {
```

Comment:
I think checking for specific notification type isn't really worth it, we can simply just fallback to cover image if user avatar is unset.